src/main/java/util/commands/AddCommand.java

```diff
@@ -0,0 +1,40 @@
+package util.commands;
+
+import util.commons.Messages;
+import util.tasks.Task;
+import util.tasks.TaskList;
+import util.ui.Ui;
+/**
+ * Command representing the addition of a
+ * task to the task list.
+ *
+ */
```

Comment:
You could consider removing the empty lines before and after your Javadocs since it's not necessary.